src/components/icons/searchIcon.tsx

```diff
@@ -18,7 +18,7 @@ function Search({ className, ...props }: SvgComponentProps) {
         fillRule='evenodd'
         clipRule='evenodd'
         d='M12.26 10.27L17.99 16l-1.49 1.49-5.73-5.73C9.7 12.53 8.41 13 7 13a6.5 6.5 0 116.5-6.5c0 1.41-.47 2.7-1.24 3.77zM7 2C4.51 2 2.5 4.01 2.5 6.5S4.51 11 7 11s4.5-2.01 4.5-4.5S9.49 2 7 2z'
-        fill='#000'
+        // fill='#000'
```

Comment:
Axo que podemos excluir essas partes que não vamos mais usar. Eu to com essa mania de comentar pra não perder, mas creio que podemos ir removendo mesmo.